src/modules/authorization/sagas.spec.ts

```diff
@@ -0,0 +1,248 @@
+import { expectSaga } from 'redux-saga-test-plan'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import { put, call, select } from 'redux-saga/effects'
+import { ContractName } from 'decentraland-transactions'
+import { ChainId } from '@dcl/schemas'
+import { fetchTransactionSuccess } from '../transaction/actions'
+import { Transaction } from '../transaction/types'
+import { waitForTx } from '../transaction/utils'
+import { getData } from './selectors'
+import { AuthorizationError } from './utils'
+import { createAuthorizationSaga } from './sagas'
+import {
+  authorizationFlowFailure,
+  authorizationFlowRequest,
+  authorizationFlowSuccess,
+  fetchAuthorizationsFailure,
+  fetchAuthorizationsRequest,
+  fetchAuthorizationsSuccess,
+  grantTokenFailure,
+  grantTokenRequest,
+  grantTokenSuccess,
+  revokeTokenFailure,
+  revokeTokenRequest,
+  revokeTokenSuccess
+} from './actions'
+import { Authorization, AuthorizationAction, AuthorizationType } from './types'
+
+const authorizationSaga = createAuthorizationSaga()
+
+const authorization: Authorization = {
+  type: AuthorizationType.ALLOWANCE,
+  address: '0x9c76ae45c36a4da3801a5ba387bbfa3c073ecae2',
+  contractAddress: '0x9c76ae45c36a4da3801a5ba387bbfa3c073ecae3',
+  authorizedAddress: '0x9c76ae45c36a4da3801a5ba387bbfa3c073ecae4',
+  contractName: ContractName.MANAToken,
+  chainId: ChainId.ETHEREUM_GOERLI
+}
+
+describe('handleAuthorizationFlowRequest', () => {
+  describe('when granting a token', () => {
+    describe('and authorization flow finishes successfully', () => {
+      it('should put success action', () => {
```

Comment:
If I'm not wrong, as we want to refer to a specific action (the success one) we must use `the`.
```suggestion
      it('should put the success action', () => {
```
If that's correct, the same would apply to the other describes mentioning an specific action.